simulator/vehicles/iris-ap/models/iris_ap/model.sdf.xacro

```diff
@@ -27,6 +28,79 @@
       </axis>
     </joint>
 
+    <link name="hokuyo_link">
+      <pose>0 0 0 0 1.57075 0</pose>
+      <inertial>
+        <mass>0.016</mass>
+        <inertia>
+          <ixx>0.0001</ixx>
+          <ixy>0</ixy>
+          <ixz>0</ixz>
+          <iyy>0.0001</iyy>
+          <iyz>0</iyz>
+          <izz>0.0001</izz>
+        <!-- low intertia necessary to avoid to disturb the drone -->
+        </inertia>
+      </inertial>
+      <!-- Visuals / Collisions omitted for this example -->
+      <sensor type="gpu_ray" name="head_hokuyo_sensor">
```

Comment:
Does gpu_ray work under windows?

Reply:
unsure, need to test. Might require changing to `ray` as its only a 1D `ray` anyway. I will just change to ray simplicity